pydicom/dataset.py

```diff
@@ -961,14 +961,7 @@ def _get_PIL_supported_compressed_pixeldata(self):
                     try:
                         decompressed_image = PILImg.open(fio)
                     except IOError as e:
-                        try:
```

Comment:
These lines are from #351  to address issue #331 .  In the comment chain for issue #331 , I lamented the ugliness of the change, but could not think of anything better.  Reverting this should regress issue #331 for python3